great_ape_safe/api_safe/opolis.py

```diff
@@ -0,0 +1,24 @@
+from brownie import interface
+
+
+class Opolis():
+    def __init__(self, safe):
+        self.safe = safe
+        # have to use erc20 interface for proxy token
+        self.work = interface.IERC20Metadata("0x6002410dDA2Fb88b4D0dc3c1D562F7761191eA80")
+        self.staking_helper = safe.contract("0x8bF5aD0dBa1e29741740D96E55Bf27Aec30B18E2")
+        self.whitelist = safe.contract("0x44a0487656420FDc15f9CA76dd95F3b8a2ef0Baa")
```

Comment:
for this repo, we decided to restrict 0x addresses to the `addresses.py` file only, to make sure no malicious contracts or wallets sneak in. it just makes reviewing a bit secure.

could you move them there?

Reply:
will move all addresses to `addresses.py` as mentioned in #15 in this pr